rts/Rendering/Env/Particles/ProjectileDrawer.cpp

```diff
@@ -676,14 +676,28 @@ void CProjectileDrawer::DrawProjectileShadow(CProjectile* p)
 
 void CProjectileDrawer::DrawProjectilesMiniMap()
 {
-	for (int modelType = MODELTYPE_3DO; modelType < MODELTYPE_CNT; modelType++) {
-		const auto& mdlRenderer = modelRenderers[modelType];
-		// const auto& projBinKeys = mdlRenderer.GetObjectBinKeys();
+	{
+		ZoneScopedN("DrawProjectilesMiniMap::ModelProjectiles");
+		for (int modelType = MODELTYPE_3DO; modelType < MODELTYPE_CNT; modelType++) {
+			const auto& mdlRenderer = modelRenderers[modelType];
+			// const auto& projBinKeys = mdlRenderer.GetObjectBinKeys();
 
-		for (unsigned int i = 0, n = mdlRenderer.GetNumObjectBins(); i < n; i++) {
-			const auto& projectileBin = mdlRenderer.GetObjectBin(i);
+			for (unsigned int i = 0, n = mdlRenderer.GetNumObjectBins(); i < n; i++) {
+				const auto& projectileBin = mdlRenderer.GetObjectBin(i);
```

Comment:
Are these two light green highlights above spaces?

Reply:
No, it's tabs